kie-archetypes/kie-kjar-archetype/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -0,0 +1,47 @@
+println "[Running post-generation script]";
+
+def caseProject = "${caseProject}";
+def moduleDir = new File(request.getOutputDirectory()+"/"+request.getArtifactId());
+
+if( caseProject == "true" ) {
+    println "Updating case project...";
+    def baseProjectDDFile = new File(moduleDir, "src/main/resources/META-INF/kie-deployment-descriptor.xml");
+    boolean baseProjectDDFileDeleted =  baseProjectDDFile.delete();
```

Comment:
any chances of deleting a descriptor that the user has customized, losing the changes?